deploy/modules/share/tidb-cluster-release/main.tf

```diff
@@ -22,6 +23,7 @@ data "helm_repository" "pingcap" {
 }
 
 resource "helm_release" "tidb-cluster" {
+  count = var.create ? 1 : 0
```

Comment:
The '=' should be aligned with others in the same block.
Same comments for the other updates in this file.

Reply:
I think we need to add Terraform `fmt` and `validate` to CI.

Reply:
There is already an issue tracking this https://github.com/pingcap/tidb-operator/issues/823

Reply:
if there's a blank line between, then the `=` don't have to be aligned